### DISCUSSION TITLE
Consistent styles - main page headings

### DISCUSSION THREAD

**Discussion by MichaelKelly9** [MichaelKelly9/Tafe24S1T104_Ext#6] [General]
Hi @ZIHAOJIA 

We should decide on a specific Font size, color and style for the main page headings e.g. Currency Conversion Calculator, Mortgage Calculator etc.

The main menu heading is currently 45 font size and blue foreground, would you be happy with those settings?

**Comment by ZIHAOJIA** [comment]
all right, font size color is ok. So we donnot need to modify on dev branch, we just fix it up on each F branch.i will finish them tomorrow. thanks Michael.
